waku/v2/rendezvous/rendezvous.go

```
@@ -56,14 +56,19 @@ func (r *Rendezvous) SetHost(h host.Host) {
 }
 
 func (r *Rendezvous) Start(ctx context.Context) error {
-	return r.CommonService.Start(ctx, r.start)
+	return r.CommonDiscoveryService.Start(ctx, r.start)
 }
 
 func (r *Rendezvous) start() error {
-	err := r.db.Start(r.Context())
-	if err != nil {
-		return err
+	if r.db != nil {
+		if err := r.db.Start(r.Context()); err != nil {
+			return err
+		}
```

Maybe we should fail the start in case db or peerConnector is not set.

---

Ok, then it is fine.